### ISSUE TITLE
Config Error with Orange OMEC Teleport

### ISSUE THREAD

**Issue by dhowe** [monocasual/giada#319]
Greetings,
Trying to use Giada with the Orange OMEC Teleport interface and can't get a working input.
- OS X 10.13.6  (CoreAudio)
- Giada 0.16.1 (1.0)

When I start Giada I see the 'Your soundcard isn't conifgured correctly' alert. 

 I believe the correct device is the 'BurrBrown from Texas Instruments: USB AUDIO  CODEC' however when I select it I get zero inputs. I also notice that it shows up twice in the logs, but only once in the config interface (see images below).

Giada Log:
```
[init] MIDI map read failed!
[KA] using system 0x10
[KA] Opening devices 1 (out), 3 (in), f=44100...
[KA] 8 device(s) found
  0) Apple Inc.: Built-in Microphone
  1) Apple Inc.: Built-in Output
  2) Apple Inc.: Display Audio
  3) BurrBrown from Texas Instruments: USB AUDIO  CODEC
  4) BurrBrown from Texas Instruments: USB AUDIO  CODEC
  5) ma++ ingalls for Cycling '74: Soundflower (2ch)
  6) ma++ ingalls for Cycling '74: Soundflower (64ch)
  7) zoom.us: ZoomAudioDevice
[KA] rtSystem init error: RtApiCore::probeDeviceOpen: the device (3) does not support the requested channel count.
[mixer::init] buffers ready - framesInSeq=88200, framesInBuffer=32
[KM] using system 0x1
[KM] 0 output MIDI ports found
[KM] 0 input MIDI ports found
[init] All subwindows and UI thread closed
[init] configuration saved
[init] PluginHost cleaned up
[init] Giada 0.16.1 closed
```

![Screen Shot 2020-03-11 at 6 35 15 PM](https://user-images.githubusercontent.com/737638/76409517-6c040300-63c9-11ea-9677-ebcbe7ae4079.png)
![Screen Shot 2020-03-11 at 6 34 08 PM](https://user-images.githubusercontent.com/737638/76409520-6d353000-63c9-11ea-8730-02cb4a2944c8.png)
![Screen Shot 2020-03-11 at 6 33 57 PM](https://user-images.githubusercontent.com/737638/76409527-6efef380-63c9-11ea-8bbe-dbe2a29fd329.png)

**Comment by dhowe** [comment]
perhaps related to #298

**Comment by monocasual** [comment]
This (and maybe also #298) is 100% related to #119: your device has a single-channel (mono) input and currently Giada wants stereo pairs only. The problem will be addressed soon, before releasing version 1.0. Thanks for your patience :)

**Comment by dhowe** [comment]
Thanks. Is there a target date for release of 1.0 ?

**Comment by monocasual** [comment]
> Thanks. Is there a target date for release of 1.0 ?

Hopefully (!) by the end of this year. Anyway the mono I/O support will be added way sooner, once the 0.16 branch is done.

**Comment by dhowe** [comment]
Great, thanks... excited to try it

**Comment by monocasual** [comment]
Closing as a duplicate of #157 . Should have been fixed with 75f716c.
